docs/real-time-analytics/mainframe-offload/process.sql

```diff
@@ -0,0 +1,35 @@
+SET 'auto.offset.reset' = 'earliest';
+
+-- Create stream of transactions from the Kafka topic
+CREATE STREAM mq_transactions (
+  dep_account_no STRING,
+  dep_balance_dollars BIGINT,
+  dep_balance_cents BIGINT,
+  timestamp BIGINT
+) WITH (
+  KAFKA_TOPIC = 'mq_transactions',
+  VALUE_FORMAT = 'JSON',
+  PARTITIONS = 6
+);
+
+-- Normalize the data and calculate timestamp deltas
+CREATE STREAM mq_transactions_normalized WITH (KAFKA_TOPIC = 'mq_transactions_normalized') AS SELECT
+  dep_account_no,
+  CAST(CONCAT(CAST(dep_balance_dollars AS STRING),'.',CAST(dep_balance_cents AS STRING)) AS DOUBLE) AS BALANCE,
+  timestamp AS ts_stream, 
+  UNIX_TIMESTAMP() AS ts_cache,
+  (UNIX_TIMESTAMP() - timestamp) AS ts_delta
+FROM mq_transactions
+EMIT CHANGES;
```

Comment:
The formatting of these two `CREATE`s do not seem consistent in their handling of the `WITH` clause and properties.  Also the `AS SELECT` seems inconsistent with others, for example : customer 360 loyalty has some CSAS

Reply:
@rspurgeon modified formatting in latest commit, WDYT?